src/transformers/models/bert/configuration_bert.py

```diff
@@ -140,3 +141,4 @@ def __init__(
         self.initializer_range = initializer_range
         self.layer_norm_eps = layer_norm_eps
         self.gradient_checkpointing = gradient_checkpointing
+        self.attention_head_size = attention_head_size
```

Comment:
This is an attribute that would definitely be misleading for other architectures, as we would expect this to be a configurable attribute when it doesn't actually affect any other architectures than the span classification architecture.

Reply:
This is not specific to span classification architecture. Any attention head pruned architectures need this. This is because the head pruned architectures doesn't hold the assumption (num_heads * head_size = hidden_size). In the current implementation, `head_size` is calculated by `hidden_size / num_heads`.

Reply:
@ykim362 I understand why you want to add this parameter, but `transformers` is a bit special in the sense that it computes a single `attention_head_size` from `num_attention_heads` and hidden_size`. Both having `attention_head_size` as a parameter or having `hidden_size` and `num_attention_heads` as parameters to define attention_head_size is fine, but we cannot have both. We decided to not have `attention_head_size` and therefore we should absolutely not add a new `attention_head_size` parameter because this a) breaks backwards compatibility and b) leads to a lot of confusing for our users that are used to `num_attention_heads` and `hidden_size`. Hope that's fine for you :-)